orchagent/pfcwdorch.cpp

```diff
@@ -525,6 +525,10 @@ void PfcWdSwOrch<DropHandler, ForwardHandler>::registerInWdDb(const Port& port,
                 PfcWdOrch<DropHandler, ForwardHandler>::getCountersTable(),
                 sai_serialize_object_id(queueId));
     }
+
+    // Create egress table group for each port of pfcwd's interest
+    sai_object_id_t group_member_oid;
+    gPortsOrch->bindAclTable(port.m_port_id, SAI_NULL_OBJECT_ID, group_member_oid, ACL_STAGE_EGRESS);
```

Comment:
this is not correct. you want to reuse bindAclTable function, but you return false if table_oid == NULL. 
You want to create an empty table group, the return should be success, not false. 

I suggest you to define a new function createAclTableGroup(), and call gPortsOrch->createAclTableGroup(port.m_port_id, ACL_STAGE_EGRESS)

Then, you rewrite bindAclTable to use createAclTableGroup.